lib/features/modeling/cmd/AddLaneHandler.js

```diff
@@ -99,3 +100,17 @@ AddLaneHandler.prototype.preExecute = function(context) {
     height: 120
   }, laneParent);
 };
+
+AddLaneHandler.prototype.postExecute = function(context) {
+
+  // (3) Re-layout participant's connections.
+  var modeling = this._modeling,
+      shape = context.shape,
+      participant = getLanesRoot(shape),
+      connections = participant.incoming.concat(participant.outgoing);
+
+
+  forEach(connections, function(c) {
+    modeling.layoutConnection(c);
```

Comment:
Source changed / target changed hints may be relevant here. Did you check that?

Reply:
First it seemed unnecessary, but then I just discovered an issue with participants placed next to the resized one. I will have a fix for that.